### ISSUE TITLE
Is there a way to change the json name of the columns?

### ISSUE THREAD

**Issue by abilash13** [volatiletech/sqlboiler#584]
Is there a way we can provide aliases for the json names in the table struct tags?

For example, the column name is Incoming_Email_c.

(this is the generated code)
``IncomingEmailC               null.String       `boil:"Incoming_Email__c" json:"Incoming_Email__c,omitempty" toml:"Incoming_Email__c" yaml:"Incoming_Email__c,omitempty"``

Is there a way to convert it to incomingEmail?

(so that the generated code looks like this)
``IncomingEmail               null.String       `boil:"Incoming_Email__c" json:"incomingEmail,omitempty" toml:"incomingEmail" yaml:"incomingEmail,omitempty"``

I know we can use aliases in the config to change the name of the column inside the struct. But I need to change it in the json (and all the other types) tags as well.

For context, we have a legacy database and we can't really change the names to saner ones.

PS: Thanks for the great work on this tool!

**Comment by aarondl** [comment]
Hmm. Essentially a feature request that uses aliases to calculate the struct tags is what's being asked for then?

**Comment by abilash13** [comment]
Yup you are right. Either two kinds of aliases (one for the Struct and one for the Struct Tags). Or the same one for both. Whichever is easier or makes more sense.

**Comment by aarondl** [comment]
It's a tough feature, because some people with more sane names may not want their JSON changing and they've only changed column names to avoid conflicts or what have you. Hard to make a decision on that.

**Comment by abilash13** [comment]
Cool I understand. For now I am overriding the default template file (00_struct.go.tpl), and changing the columns names to the column aliases. But because of this, I have to include all the other default template files in the repo as well.

Is this the only workaround available, or do you happen to know a cleaner one?

**Comment by aarondl** [comment]
None that I'm aware of. This seems fine :)
